docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+services:
+
+  nginx:
+    restart: always
+    build:
+      context: ./docker/nginx_custom/
+      dockerfile: Dockerfile
+    links:
+      - web:web
+    ports:
+      - "80:80"
+
+  web:
+    build: .
```

Comment:
Why is the web container not marked as "restart: always"? I've noticed this elsewhere too. Same goes for other docker images...